tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -31,7 +32,19 @@ class ExpectSuite extends FunSuite {
   }
   test("metacp.expect") {
     import MetacpExpect._
-    this.assertNoDiff(loadObtained, loadExpected)
+
+    val javaVersion = Properties.javaVersion.takeWhile(_.isDigit).toInt
+    val expected =
+      if (javaVersion >= 17)
+        // strictfp is deprecated since JDK 17 and therefore ACC_STRICT is no longer emitted.
+        // See https://openjdk.org/jeps/306.
+        loadExpected.replace(
+          """|com/javacp/Test#strictfpMethod(). => @strictfp private[javacp] method strictfpMethod(): Unit
+             |  strictfp => scala/annotation/strictfp#""".stripMargin,
+          "com/javacp/Test#strictfpMethod(). => private[javacp] method strictfpMethod(): Unit".stripMargin
```

Comment:
since floating-point operations are all strict when running JDK17+ bytecode, we could always emit the flag instead of relying on the now deprecated `ACC_STRICT` in order to keep the semantics, but that would be verbose for all tests

Reply:
I think we can skip it then as it's not something I have ever seen used by the tooling using scalameta

Reply:
It was never implemented in https://github.com/sourcegraph/scip-java/issues/144. I just added a note in the docs.